BoundingBoxes[] boundingBoxesVector

sensor_msgs/Image image